src/libs/storage.js

```diff
@@ -65,5 +66,13 @@ export const Storage = {
 
   removeData: key => {
     sessionStorage.removeItem(key);
+  },
+
+  setEnv: (value) => {
+    sessionStorage.setItem(ENV, value);
+  },
+
+  getEnv: () => {
+    return sessionStorage.getItem(ENV);
```

Comment:
nice. i ran into an issue like this before and didn't how to handle it best - i like this solution.